### PR TITLE
should there be a problem with the sddm user, be more verbose instead of normal exit

### PR DESCRIPTION
**Pull request by mathieujobin**
fix in relation to issue #371 

print verbose messages in case there is a  problem with sddm user.
also, do failure exit instead of normal exit

cheers,

### CODE REVIEW

File: src/helper/UserSession.cpp [hunk 1]
```diff
@@ -61,21 +61,28 @@ namespace SDDM {
...
-        struct passwd *pw = getpwnam(qobject_cast<HelperApp*>(parent())->user().toLocal8Bit());
-        if (setgid(pw->pw_gid) != 0)
-            bail(2);
-        if (initgroups(pw->pw_name, pw->pw_gid) != 0)
-            bail(2);
-        if (setuid(pw->pw_uid) != 0)
-            bail(2);
-        if (chdir(pw->pw_dir) != 0)
-            bail(2);
+        //TODO: why do we need to fetch the info from the passwd file?
```
Comment (davidedmundson) [inline src/helper/UserSession.cpp]: Remove this line.
Code is not a bug tracker

**Comment by davidedmundson** [comment]
Looks good. Make that one change I want and I'll merge this.

Thanks.

**Comment by mathieujobin** [comment]
i removed that comment and fix a syntax error I had.

**Comment by davidedmundson** [comment]
Thanks very much.

**Pull request merged**